### PR TITLE
Verification revoke status check added

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/apollo/gql/gqlUser.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ export const FETCH_USER_PROJECTS = gql`
 				totalDonations
 				totalReactions
 				verified
+				verificationStatus
```
Comment (RamRamez) [inline src/apollo/gql/gqlUser.ts:53]: Is `verificationStatus` on `projectsByUserId` used anywhere?

File: src/apollo/types/types.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ export interface IProject {
 	impactLocation?: string;
 	qualityScore?: number;
 	verified?: boolean;
+	verificationStatus?: string;
```
Comment (RamRamez) [inline src/apollo/types/types.ts:21]: I think you can change `string` with `EProjectVerificationStatus`

**Review by RamRamez — CHANGES_REQUESTED** [review]
Thanks @mateodaza, great. Just a minor fix.

File: src/apollo/gql/gqlUser.ts (hunk 1 above)
Comment (mateodaza) [reply src/apollo/gql/gqlUser.ts:53]: Not at the moment but it's still an important property of the schema. You want me to remove it?

**Review by RamRamez — APPROVED** [review]
Thanks @mateodaza

**Pull request merged**